lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -102,6 +102,7 @@ public function GetQueue($id, $short = false)
      *          all - filter is off
      *          -1 - without verifier set
      *          -2 - with verifier set
+     *      rights - tickets from queues with particular permissions (default:null whitch means tickets from all queues),
```

Comment:
Co oznacza to nowe ustawienie? Z tego co napisałeś po angielsku wynika, że ma jakiś związek z uprawnieniami, ale jaki i z którymi?